apps/hash-frontend/src/pages/drafts.page/draft-entity/draft-entity-action-buttons.tsx

```diff
@@ -21,7 +22,53 @@ import {
 } from "../../../graphql/queries/knowledge/entity.queries";
 import { useDraftEntities } from "../../../shared/draft-entities-context";
 import { CheckRegularIcon } from "../../../shared/icons/check-regular-icon";
+import { FeatherRegularIcon } from "../../../shared/icons/feather-regular-icon";
 import { Button } from "../../../shared/ui";
+import { LinkLabelWithSourceAndDestination } from "../../shared/link-label-with-source-and-destination";
+
+const LeftOrRightEntityEndAdornment: FunctionComponent<{
```

Comment:
@CiaranMn I realised too late in development that we may want to make this re-usable within the `LinkLabelWithSourceAndDestination` component so that we can display whether the source or destination is a draft in the entity type editor as well. If we do want this, will refactor that in follow-up.